io-engine/src/core/env.rs

```diff
@@ -561,7 +565,7 @@ impl MayastorEnvironment {
     }
 
     /// initialize the logging subsystem
-    fn init_logger(&mut self) -> Result<()> {
+    fn init_logger(&mut self) {
```

Comment:
Any reason for not making failure an error now?

Reply:
This result was never checked in a meaningful way: it was just unwraped, causing panic if spdk_log_set_flag() failed.